doc/source/scalability/innodb_adaptive_hash_index_partitions.rst

```diff
@@ -4,7 +4,7 @@
  Multiple Adaptive Hash Search Partitions
 ==========================================
 
-The |InnoDB| adaptive hash index can have contention issues on multi-core systems when you run a mix of read and write queries that need to scan secondary indexes. This feature splits the adaptive hash index across several partitions to avoid such problems.
+The |InnoDB| adaptive hash index can have contention issues on multi-core systems when you run a mix of read and write queries that need to scan secondary indexes. This feature splits the adaptive hash index across several equal-sized partitions to avoid such problems.
```

Comment:
Just saying "equal-sized" does not explain why this might be bad: see comment 1 on the bug. Each index is assigned to some particular partition, thus some partitions could be very hot and some completely unused